src/main/java/io/jenkins/plugins/synopsys/security/scan/factory/ScanParametersFactory.java

```diff
@@ -228,6 +228,10 @@ public static Map<String, Object> prepareBlackDuckParametersMap(SecurityScan sec
                     ApplicationConstants.BLACKDUCK_DOWNLOAD_URL_KEY, securityScan.getBlackduck_download_url());
         }
 
+        if (!Utility.isStringNullOrBlank(securityScan.getProject_directory())) {
+            blackDuckParameters.put(ApplicationConstants.PROJECT_DIRECTORY_KEY, securityScan.getProject_directory());
```

Comment:
This is not covered by unit test.

Check this PR file changes, github is showing this warning in many places. Check if all cases are covered in test.